### PR TITLE
k8s_scale - add option label_selectors and continue_on_error

### PR DESCRIPTION
**Pull request by abikouo**
##### SUMMARY

``k8s_scale`` - add new parameter ``label_selectors`` to support scale for multiple resources, and new parameter ``continue_on_error`` to determine wether to continue with others resources when an error occurs.

#112 
##### ISSUE TYPE

- Feature Pull Request
##### COMPONENT NAME

k8s_scale
##### ADDITIONAL INFORMATION

```paste below
- kubernetes.core.k8s_scale:
    kind: Deployment
    namespace: default
    label_selectors:
      - app = test
    replicas: 3
```

### CODE REVIEW

File: changelogs/fragments/114-k8s_scale-add-label-selectors-and-continue-on-error.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+minor_changes:
+  - k8s_scale - ability to scale multiple resource using ``label_selectors`` (https://github.com/ansible-collections/community.kubernetes/pull/114).
+  - k8s_scale - new parameter to determine wether to continue or not on error when scaling multiple resources (https://github.com/ansible-collections/community.kubernetes/pull/114).
```
Comment (Akasurde) [inline changelogs/fragments/114-k8s_scale-add-label-selectors-and-continue-on-error.yaml:4]:
```suggestion
  - k8s_scale - new parameter to determine whether to continue or not on error when scaling multiple resources (https://github.com/ansible-collections/community.kubernetes/pull/114).
```

File: molecule/default/tasks/scale.yml [hunk 2]
```diff
@@ -202,6 +202,69 @@
...
+      kubernetes.core.k8s_info:
+        kind: Deployment
+        namespace: "{{ scale_namespace }}"
+        label_selectors:
+          - app=nginx
+      register: resource
+
+    - assert:
+        that:
+          - resource.resources | list | length == 2
```
Comment (Akasurde) [inline molecule/default/tasks/scale.yml:221]:
```suggestion
          - resource.resources | list | length == 2
          - resource is changed
```

File: plugins/modules/k8s_scale.py [hunk 3]
```diff
@@ -23,13 +23,27 @@
...
 
 extends_documentation_fragment:
   - kubernetes.core.k8s_name_options
   - kubernetes.core.k8s_auth_options
   - kubernetes.core.k8s_resource_options
   - kubernetes.core.k8s_scale_options
 
+options:
+  label_selectors:
+    description: List of label selectors to use to filter results
```
Comment (Akasurde) [inline plugins/modules/k8s_scale.py:36]:
```suggestion
    description: List of label selectors to use to filter results.
```

File: plugins/modules/k8s_scale.py [hunk 4]
```diff
@@ -82,6 +96,15 @@
...
     replicas: 3
     wait: no
+
+- name: Scale deployment using label selectors (continue operation in case error occured on one resource)
+  kubernetes.core.k8s_scale:
+    replicas: 3
+    kind: Deployment
+    namespace: test
+    label_selectors:
+      - app = test
```
Comment (Akasurde) [inline plugins/modules/k8s_scale.py:106]: Since this is string I would avoid spaces in between 

```suggestion
      - app=test
```

**Review by Akasurde — CHANGES_REQUESTED** [review]
Rest LGTM

**Review by gravesm — APPROVED** [review]

**Pull request merged**